container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -34,7 +34,22 @@ TEMPEST_ARGS=""
 [[ ! -z ${TEMPESTCONF_FLAVOR_MIN_DISK} ]] && TEMPESTCONF_ARGS+="--flavor-min-disk ${TEMPESTCONF_FLAVOR_MIN_DISK} "
 [[ ! -z ${TEMPESTCONF_NETWORK_ID} ]] && TEMPESTCONF_ARGS+="--network-id ${TEMPESTCONF_NETWORK_ID} "
 
+TEMPESTCONF_OVERRIDES="$(echo ${TEMPESTCONF_OVERRIDES} | tr '\n' ' ') identity.v3_endpoint_type public"
```

Comment:
is this change required? The line's been moved from line 61 to 37

Reply:
Not really. But I'm trying to follow the pattern of having all the one-liners that depend on external variables at the beginning of the script. Should I move it back?

Reply:
nah, it's fine